app/src/main/java/com/thinqtv/thinqtv_android/MainActivity.java

```diff
@@ -205,6 +203,14 @@ public void setUpcomingEvents(String fullEventsJSON)
                 newEvent_time.setPadding(750, 80, 0, 0);
                 newEvent_time.setTextColor(getResources().getColor(R.color.colorPrimary));
 
+                // formats the date from above into viewable format (BUT NOW ITS START TIME)
+                displayFormat = new SimpleDateFormat("hh:mm");
```

Comment:
Would it be possible to add AM/PM? Right now an event at like 8:00 is a little ambiguous.